services/federator/src/Federator/Response.hs

```diff
@@ -145,7 +149,7 @@ type AllEffects =
      ServiceStreaming,
      Input RunSettings,
      Input Http2Manager, -- needed by Remote
-     Input FederationDomainConfigs, -- needed for the domain list.
+     Input FedUp.FederationDomainConfigs, -- needed for the domain list and federation policy.
```

Comment:
😂

Reply:
Probably part of a pile of changes I made and then backed out.